package.json

```diff
@@ -39,49 +39,49 @@
   },
   "homepage": "https://github.com/makeomatic/redux-connect",
   "peerDependencies": {
-    "prop-types": "16.x.x",
+    "prop-types": "15.x.x",
     "react": "16.x.x",
```

Comment:
i'd assume to tighten react peer dep for a bit as react-redux@7 requires ^16.8.4

Reply:
Agree, I'd go with latest react if possible